using.regex.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+text = """Из 35 футболистов, забивших как минимум 7 голов на чемпионатах мира, только у троих футболистов средний 
+        показатель превышает 2 гола за игру. Эти 35 игроков представляют 14 футбольных сборных"""
+
+
+def multiplied(matchobj):
+    number = matchobj.group(0)
+    multiplied_number = int(number) * 2
+    string_number = str(multiplied_number)
+    return string_number
+
+
+final_text = re.sub(r'\b\d+\b', multiplied, text)

```

Comment:
it will be better to get rid of the usual function in place of **repl** param. 
Could you please try to do it using the lambda function?

Reply:
Done